app/controllers/support_interface/zendesk_controller.rb

```diff
@@ -36,6 +38,19 @@ def destroy
       end
     end
 
+    def export
+      @zendesk_delete_requests = ZendeskDeleteRequest.all
+      respond_to do |format|
+        format.csv do
+          response.headers["Content-Type"] = "text/csv"
+          response.headers[
+            "Content-Disposition"
+          ] = "attachment; filename=zendesk_delete_requests.csv"
```

Comment:
`ZendeskDeleteRequest` is our internal name, but for our users, something like this would be more intuitive:

```suggestion
          ] = "attachment; filename=deleted_zendesk_tickets.csv"
```

Reply:
Maybe also prefix a timestamp with an ISO8166 date to the filename.